server/views/pages/work.njk

```diff
@@ -28,65 +28,69 @@
     <div class="container">
       <div class="grid">
         <div class="{{ {s: 12, m: 7, l: 8, xl: 7, shiftXl: 1} | gridClasses }} {{ {s:4} | spacingClasses({margin: ['bottom']}) }}">
-          <h1 id="work-info" class="{{ {s:'HNM3', m:'HNM2', l:'HNM1'} | fontClasses }} {{ {s:0} | spacingClasses({margin: ['top']}) }}">{{work.title}}</h1>
-
-          {% if work.creators.length > 0 %}
-            {% set needsDivider = work.createdDate.label or work.genres.length > 0 or work.subjects.length > 0 or work.lettering > 0 or work.descriptionArray.length > 0 %}
-            {% componentV2 'meta-unit', {headingText: 'By', content: work.creators | getLinkObjects('label', 'creators:'), includeDivider: needsDivider} %}
-          {% endif %}
-
-          {% if work.createdDate.label %}
-            {% set needsDivider = work.genres.length > 0 or work.subjects.length > 0 or work.lettering.length > 0 or work.descriptionArray.length > 0 %}
-            {% componentV2 'meta-unit', {headingText: 'Date', content: work.createdDate.label, includeDivider: needsDivider} %}
-          {% endif %}
-
-          {% if work.genres.length > 0 %}
-            {% set needsDivider = work.subjects.length > 0 or work.lettering.length > 0 or work.descriptionArray.length > 0 %}
-            {% componentV2 'meta-unit', {headingText: 'Genre', content: work.genres | getLinkObjects('label'), includeDivider: needsDivider} %}
-          {% endif %}
-
-          {% if work.subjects.length > 0 %}
-            {% set needsDivider = work.lettering.length > 0 or work.descriptionArray.length > 0 %}
-            {% componentV2 'meta-unit', {headingText: 'Subject', content: work.subjects | getLinkObjects('label'), includeDivider: needsDivider} %}
-          {% endif %}
-
-          {% if work.lettering.length > 0 %}
-            {% set needsDivider = work.descriptionArray.length > 0 %}
-            {% componentV2 'meta-unit', {headingText: 'Lettering', content: work.lettering, includeDivider: needsDivider} %}
-          {% endif %}
-
-          {% if work.descriptionArray.length > 0 %}
-            <div>
-              <h2 class="{{ {s:'HNM5' , m:'HNM4'} | fontClasses }} {{ {s:0} | spacingClasses({margin: ['top']}) }} {{ {s:1} | spacingClasses({margin: ['bottom']}) }}">
-                Description
-              </h2>
-              {% for paragraph in work.descriptionArray %}
-              <p class="{{ {s:'HNL5' , m:'HNL4'} | fontClasses }} {{ {s:2} | spacingClasses({margin: ['bottom']}) }}">{{paragraph}}</p>
-              {% endfor %}
-            </div>
-          {% endif %}
+          <div class="{{ {s: 5} | spacingClasses({margin: ['bottom']}) }}">
```

Comment:
This change (lines 31–75) is just the addition of a wrapping div to add bottom-margin

Reply:
https://github.com/wellcometrust/wellcomecollection.org/pull/1859/files?w=1